package.json

```diff
@@ -28,47 +29,45 @@
     "url": "https://github.com/DomainGroupOSS/react-media-query-hoc/issues"
   },
   "homepage": "https://github.com/DomainGroupOSS/react-media-query-hoc#readme",
-  "eslintConfig": {
-    "extends": "airbnb",
-    "rules": {
-      "react/jsx-filename-extension": "off"
-    }
-  },
   "babel": {
     "presets": [
-      "es2015",
-      "stage-0",
-      "react"
+      "react-app"
+    ],
+    "plugins": [
+      "transform-es2015-modules-commonjs",
+      "transform-runtime"
     ]
   },
   "peerDependencies": {
-    "react": "^15.1.0"
+    "react": "^15.1.0 || ^16.0.0"
   },
   "devDependencies": {
     "babel-cli": "^6.10.1",
-    "babel-preset-es2015": "^6.9.0",
-    "babel-preset-react": "^6.5.0",
-    "babel-preset-stage-0": "^6.5.0",
+    "babel-plugin-transform-es2015-modules-commonjs": "^6.26.2",
```

Comment:
why was this changed ? isnt it easier to just use `babel-preset-es2015` so we dont need to worry about individual babel plugins

Reply:
This was because `babel-preset-react-app` does not transpile CJS modules, because Webpack supports ES modules. But I have actually decided to use the env and react preset instead after taking a closer look at `babel-preset-react-app` src, it's more geared towards an app vs a library e.g it removes proptype warnings for prod.

But yearly presets are not recommended:

> Each yearly preset only compiles what was ratified in that year. babel-preset-env replaces es2015, es2016, es2017 and latest

https://babeljs.io/docs/plugins/

> Without any configuration options, babel-preset-env behaves exactly the same as babel-preset-latest (or babel-preset-es2015, babel-preset-es2016, and babel-preset-es2017 together).

https://babeljs.io/docs/plugins/preset-env/

Also see https://github.com/contra/react-responsive/blob/master/package.json#L77-L80